test/ipynb/mpl/__init__.py

```diff
@@ -10,7 +10,7 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Image comparison test for MPL graph drawer.
+"""Image comparison test for MPL drawer.
```

Comment:
```suggestion
"""Image comparison test for MPL circuit drawer.
```

Reply:
I think just `MPL drawer` is correct, as it tests both the graphs and the circuits, then each subfolder has it's own `__init.py__` where it specifies if it's testing the graphs or circuits

Reply:
In that case... 

```suggestion
"""Image comparison test for MPL drawers.
```